base/accumulate.jl

```diff
@@ -250,6 +262,10 @@ julia> accumulate(+, fill(1, 3, 3), dims=2)
 ```
 """
 function accumulate(op, A; dims::Union{Nothing,Integer}=nothing, kw...)
```

Comment:
Sorry that I missed that on first review: wouldn't this function need a compat annotation, like `cumsum` and `cumprod` ?

Reply:
Thanks for catching it. I added it in https://github.com/JuliaLang/julia/pull/34656/commits/7a4d7febaf219f80fbdbe4d6cb028c1ab2e2d40d.

Reply:
Thanks, in the meantime I also thought about compat for the iterator version, and was sorry to have to ask you again another change, but you already made it :)